### PR TITLE
Update single label creation workflow #83 proposal

### PR DESCRIPTION
**Pull request by ScreamingDev**
See #83

### CODE REVIEW

File: components/woo/shipping-method.php [hunk 1]
```diff
@@ -480,6 +480,71 @@ public function init_settings_fields()
...
 				'desc_tip'    => true,
 			),
+			'banking_information' => array(
+				'title'       => _x(
+					'Banking information',
+					'Backend: Title of the settings section',
+					'shipcloud-for-woocommerce'
+				),
+				'type'        => 'title',
+				'description' => sprintf( __( 'Fill in defaults for cash on deliveryŝ.', 'shipcloud-for-woocommerce' ) ),
```
Comment (andrecedik) [inline components/woo/shipping-method.php]: `deliveryŝ` <-- that's a typo right? The last character should probably not be there :grin:

**Comment by ScreamingDev** [comment]
@andrecedik please test again.
Typo is gone, 
Buttons are clickable

**Comment by andrecedik** [comment]
When you have a prepared shipment, the buttons not really make a sense. Yes you could visit the tracking page and see that nothing is going on 😁 

But, if I click the download button the order opens in a new tab again. No download is started, because there is no file to be downloaded. I think you should remove the buttons if a shipment has no shipping label associated with it. Or you could display 2 other buttons - "edit" and "create label".

**Comment by andrecedik** [comment]
Cool, the buttons make way more sense now. Is it possible to display the same buttons you're showing for shipments with shipping labels after you've created a shipping label for a shipment that previously had none and whee´re you've created one via the button? I'm currently seeing no buttons at all after clicking the "create shipment" button.

prepared_shipment_to_label

When this PR is done, you should merge its contents into #154 to be able to show a "edit shipment" button for prepared shipments 😉

**Comment by ScreamingDev** [comment]
Please review again @andrecedik .

**Comment by andrecedik** [comment]
This looks and feels amazing 😱 
You should change the base to be merged into to `release/1.6` so we have a clean release.

**Comment by ScreamingDev** [comment]
So ... We call it 1.6 and it's good to merge? 
Simple conflict here, easy to solve. Both sides are needed ;)

@andrecedik

**Comment by andrecedik** [comment]
Yes, we'll call it 1.6. It's a major UX change which is good for the users. Oh and yes - it's good to merge @ScreamingDev 👍

**Pull request merged**